crates/common/consensus/lean/src/block.rs

```diff
@@ -36,7 +45,7 @@ pub struct Block {
 
 /// Represents a block header in the Lean chain.
 ///
-/// See the [Lean specification](https://github.com/leanEthereum/leanSpec/blob/main/docs/client/containers.md#blockheader)
+/// See the [Lean specification](https://github.com/leanEthereum/leanSpec/blob/5da200c13f5eeda0b4139b1d55970d75c011d4b2/src/lean_spec/subspecs/containers/block/block.py#L36)
```

Comment:
we should remove the link to the specification as this is inherently buggy and won't be maintained, etc etc